invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/ShareModal.js

```diff
@@ -5,59 +5,90 @@
 // Invenio RDM Records is free software; you can redistribute it and/or modify it
 // under the terms of the MIT License; see LICENSE file for more details.
 
-import React from "react";
+import React, { Component } from "react";
 import { Icon, Button, Modal, Tab } from "semantic-ui-react";
 import { i18next } from "@translations/invenio_app_rdm/i18next";
 import PropTypes from "prop-types";
 import { LinksTab } from "./AccessLinks/LinksTab";
-// import { GrantsTab } from "./GrantsTab";
+import { AccessRequestsTab } from "./AccessRequests/AccessRequestsTab";
 
-export const ShareModal = ({ record, open, handleClose, accessLinksSearchConfig }) => {
-  const panes = [
-    {
-      menuItem: { icon: "linkify", content: "Links" },
-      pane: (
-        <Tab.Pane className="borderless" attached="top" key="accessLinks">
-          <LinksTab record={record} accessLinksSearchConfig={accessLinksSearchConfig} />
-        </Tab.Pane>
-      ),
-    },
-  ];
+export class ShareModal extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      record: this.props.record,
+    };
+  }
 
-  return (
-    <Modal
-      open={open}
-      closeIcon
-      onClose={handleClose}
-      className="share-modal"
-      role="dialog"
-      aria-labelledby="access-link-modal-header"
-      aria-modal="true"
-      tab-index="-1"
-      size="large"
-      closeOnDimmerClick={false}
-    >
-      <Modal.Header id="access-link-modal-header">
-        <Icon name="share square" />
-        {i18next.t("Share access")}
-      </Modal.Header>
+  handleRecordUpdate = (updatedRecord) => {
+    this.setState({ record: updatedRecord });
+  };
 
-      <Modal.Content>
-        <Tab
-          menu={{ secondary: true, pointing: true }}
-          panes={panes}
-          renderActiveOnly={false}
-        />
-      </Modal.Content>
+  panes = (record, accessLinksSearchConfig) => {
+    return [
+      {
+        menuItem: { icon: "linkify", content: "Links" },
```

Comment:
```suggestion
        menuItem: (
           <Button
              role="tab"
              icon="linkify"
              className="item"
              id="links-tab"
              aria-controls="links-tab-panel"
              aria-selected={activeIndex === TAB_INDEX } //Set to correct value
            >
              {i18next.t("Links")}
            </Button>
        ),
```

To make the tabs accessible by keyboard (and for screen readers)